packages/ts-plugin/src/language-service/index.ts

```diff
@@ -10,19 +10,46 @@ import { decorateGetImplementation } from './implementation.js';
 import { decorateLineColumnOffset } from './line-column-offset.js';
 import { decorateRename } from './rename.js';
 
+const astroPluginPatchSymbol = Symbol('astroPluginPatchSymbol');
```

Comment:
Approved on the spot.

Reply:
Unfortunately not my code, copied that part from Svelte's TS plugin, sorry to disappoint